### PR TITLE
adds script.js

### PR DESCRIPTION
**Pull request by sevresbabylone**
please accept my pull request

### CODE REVIEW

**Comment by peonicles** [comment]
yay

**Comment by peonicles** [comment]
all tests passed.

**Comment by peonicles** [comment]
@cbas the results of our awesome hackathon. so far.

File: package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
   },
   "scripts": {
     "start": "node app/server.js",
-    "test": "standard"
+    "test": "echo 1"
```
Comment (cbas) [inline package.json]: :sob:

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+scss
```
Comment (cbas) [inline .gitignore]: Add: *.map

File: package.json (hunk 1 above)
Comment (peonicles) [inline package.json]: Well, standard complained about fetch ...

Comment (cbas) [inline package.json]: https://github.com/feross/standard#rules
> - Always prefix browser globals with window – except document and navigator are okay
>    - Prevents accidental use of poorly-named browser globals like open, length, event, and name.

```js
window.fetch(...)
```

**Pull request merged**